woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2131,9 +2131,13 @@ def test_setitem_indexed_column_on_indexed_dataframe(sample_df):
 
     assert sample_df.ww.index is None
     assert ww.is_schema_valid(sample_df, sample_df.ww.schema)
+    expected_tags = {}
+    if hasattr(sample_df.ww.logical_types['id'], 'standard_tags'):
+        expected_tags = sample_df.ww.logical_types['id'].standard_tags
```

Comment:
It's often times preferable to have our expected values be a hardcoded value, because if there was also a bug in how we got the standard tags or logical types or something, we might very well not catch it because we use very similar lines of code in the setitem iteself.

There's no hard and fast rule, but for smaller things like a set of semantic tags, we often times write it out explicitly like we do in this test: https://github.com/alteryx/woodwork/blob/main/woodwork/tests/schema/test_table_schema_init.py#L294

So in this case, I think it'd be `assert sample_df.ww['id'].ww.semantic_tags == {'numeric'}`

Reply:
Especially because we want to be testing logical types here that have standard tags. `id` will be `Integer`, so it will have the `numeric` standard tag, but unless we write that out, there's no way of seeing that as a developer reading this test.